training/08_using_git.Rmd

```diff
@@ -36,7 +36,7 @@ You should see a prompt asking you for a URL. Fill it out as shown in the image
 
 ![](images/clone_repo.jpeg)
 
-You will be prompted for your username and password, and then Git will clone the directory. The username/password you use should be the same one you use to log in when you go to http://github.nceas.ucsb.edu/KNB/arctic-data. Now you should have a directory called arctic-data in your RStudio files window.
+You will be prompted for your username and password, and then Git will clone the directory. The username/password you use should be the same one you use to log in when you go to RT or http://github.nceas.ucsb.edu/KNB/arctic-data. Now you should have a directory called arctic-data in your RStudio files window.
```

Comment:
I would further clarify this with what Erika mentioned in the issue: "depending on where the repository is you will either need to use the credentials from the public account or the enterprise account"

because there was that confusion of the different github accounts we have

Reply:
@KellyWang1126 you can continue committing 
> Add more commits by pushing to the master branch on KellyWang1126/datateam-training.

Reply:
Sure. Just for double check, the account here should be the enterprise account right?

Reply:
in this case yes

Reply:
Jeanette usually refers to it as LDAP account